packages/@okta/vuepress-theme-prose/assets/css/base/_typography.scss

```diff
@@ -129,22 +129,44 @@
   src: url("/fonts/okticon.eot?#iefix") format("embedded-opentype"), url("/fonts/okticon.woff") format("woff"), url("/fonts/okticon.ttf") format("truetype"), url("/fonts/okticon.svg#okticon") format("svg");
 }
 
+html {
+  font-size: 16px;
+}
+
 a {
   font-weight: 500;
   text-decoration: none;
-  color: cv("link");
+  color: cv("link", "base");
```

Comment:
Have they changed or removed default value?

Reply:
They have changed the default value. It was `base` earlier. Now it is `500`